components/data/librarysize/requirements.txt

```diff
@@ -0,0 +1,2 @@
+maayanlab-bioinformatics @ git+https://github.com/Maayanlab/maayanlab-bioinformatics
```

Comment:
Please ensure this requirements.txt contains all dependencies for this particular component. maayanlab-bioinformatics is not used in this one, plotly is